Dungeoneer/src/com/interrupt/dungeoneer/entities/Item.java

```diff
@@ -130,6 +142,8 @@
 	public transient String lastMeshFile = null;
 	public transient String lastTextureFile = null;
 
+	private Entity pickupHelper = null;
```

Comment:
This could be marked 'private transient' to keep it from being serialized to disk